vmdb/app/controllers/container_controller.rb

```diff
@@ -0,0 +1,360 @@
+        page << "dhxLayoutB.cells('c').expand();" #incase it was collapsed for summary screen, and incase there were no records on show_list
+        page << javascript_show("form_buttons_div")
+        page << javascript_hide_if_exists("pc_div_1")
+        locals = {:record_id => @edit[:rec_id], :action_url => action_url}
+        page.replace_html("form_buttons_div", :partial => "layouts/x_edit_buttons", :locals => locals)
+      elsif record_showing ||
+        (@pages && (@items_per_page == ONE_MILLION || @pages[:items] == 0))
+        #Added so buttons can be turned off even tho div is not being displayed it still pops up Abandon changes box when trying to change a node on tree after saving a record
+        page << javascript_hide_if_exists("buttons_on")
+        page << "dhxLayoutB.cells('a').expand();"
+        page << "dhxLayoutB.cells('c').collapse();"
+      else
+        page << javascript_hide_if_exists("form_buttons_div")
+        page << javascript_show("pc_div_1")
+        page << "dhxLayoutB.cells('a').expand();"
+        page << "dhxLayoutB.cells('c').expand();"
+      end
+
+      page << "$('#main_div').scrollTop();"  # Scroll to top of main div
+
+      # Clear the JS gtl_list_grid var if changing to a type other than list
+      if @gtl_type && @gtl_type != "list"
+        page << "if (typeof gtl_list_grid != 'undefined') gtl_list_grid = undefined;"
+      end
+
+      # Rebuild the toolbars
+      if h_buttons && h_xml
+        page << javascript_for_toolbar_reload('history_tb', h_buttons, h_xml)
+        page << javascript_show_if_exists("history_buttons_div")
+      else
+        page << javascript_hide_if_exists("history_buttons_div")
+      end
+
+      if v_buttons && v_xml
+        page << javascript_for_toolbar_reload('view_tb', v_buttons, v_xml)
+        page << javascript_show_if_exists("view_buttons_div")
+      else
+        page << javascript_hide_if_exists("view_buttons_div")
+      end
+
+      if c_buttons && c_xml
+        page << javascript_for_toolbar_reload('center_tb', c_buttons, c_xml)
+        page << javascript_show_if_exists("center_buttons_div")
+      else
+        page << javascript_hide_if_exists("center_buttons_div")
+      end
+
+      if h_buttons || c_buttons || v_buttons
+        page << "dhxLayoutB.cells('a').expand();"
+      else
+        page << "dhxLayoutB.cells('a').collapse();"
+      end
+
+      if @record
+        page << "miq_record_id = '#{@record.id}';"  # Create miq_record_id JS var, if @record is present
+      else
+        page << "miq_record_id = undefined;"  # reset this, otherwise it remembers previously selected id and sends up from list view when add button is pressed
+      end
+
+      page << "cfmeDynatree_activateNodeSilently('#{x_active_tree}','#{x_node}');" if params[:id]
+      page << "$('##{x_active_tree}box').dynatree('#{@in_a_form && @edit ? 'disable' : 'enable'}');"
+      dim_div = @in_a_form && @edit && @edit[:current] ? true : false
```

Comment:
This ternary isn't necessary.  In ruby, boolean expressions return a value, so you can to this:

```ruby
dim_div = @in_a_form && @edit && @edit[:current]
```